include/SW_datastructs.h

```diff
@@ -1000,6 +1003,24 @@ typedef struct {
 	#endif
 } SW_GEN_OUT;
 
+/* =================================================== */
+/*                    Domain struct                    */
+/* --------------------------------------------------- */
+
+typedef struct {
+	// Spacial domain information
+	char DomainType[3]; // Either "xy" or "s" (exluding '\0')
+	int nDimX,
+		nDimY,
+		nDimS;
+
+	// Temporal domain information
+	TimeInt startyr,       /* beginning year for a set of simulation run */
+			endyr,         /* ending year for a set of simulation run */
+			startstart,    /* startday in start year */
+			endend;        /* end day in end year */
+} SW_DOMAIN;
```

Comment:
in which part are you planning to code up `suids`?

Reply:
I was planning on doing it within the branch where we will read the domain netCDF. However, I realized it's best to do it in `feature_nc_createnc` because when we create the domain file, we are going to have to create/accept suids for the program to use.